src/services/fileStorage/proxy-service.js

```diff
@@ -434,7 +434,7 @@ const signedUrlService = {
 			.then(() => strategy.getSignedUrl({
 				userId: creatorId,
 				flatFileName: fileObject.storageFileName,
-				localFileName: fileObject.name,
+				localFileName: query.name || fileObject.name,
```

Comment:
🤔 Should this be sanitized somehow (possibly in the strategy)?

I.e. I am wondering about cases such as:

1. `…?name=/etc/passwd`
2. `…?name=../..`

I assume browsers should catch these, but I am not 100% sure.

Reply:
Can you explain it with more details for me? Thank you. :) @pmeinhardt

Reply:
The `name` here is the name that will be sent to the browser in the file name header (of the top of my head I don't remember which one that is). I doubt that `/` is permitted there.

Reply:
Linking to the spec here might be good, so it's clear we thought about this to anyone who comes across this code at a later point?

— https://tools.ietf.org/html/rfc6266#section-4.3